updatemetrics.sh

```diff
@@ -14,11 +14,8 @@ echo "Backing up database first ... "
 mkdir -p ~/DBBackups
 
 today=$(date +'%m-%d-%Y')
-file_name="/home/webfiles/DBBackups/libcomp-bkp-${today}.sql"
-mysqldump --no-tablespaces --no-create-info --complete-insert --skip-triggers libcomp  Domain Library Issue Metric LibraryRelease MetricsEntry Chart ProjectType TeamType PluginUser PluginUser_groups PluginUser_projects PluginUser_teams PluginFeedback WebsiteFeedback > $file_name
-
-#pull latest repo changes
-git pull
+file_name="libcomp-bkp-${today}.sql"
```

Comment:
can we at least put this in a DBBackups folder? Otherwise, it will be lots of files in the current directory

Reply:
@XiaoleZ  So I guess based on our conversation, you should revert back all changes related to `updatemetrics.sh` (i.e., it should be the original one before your changes since you won't be dealing with it)